plugins/table.md

```diff
@@ -83,6 +83,28 @@ tinymce.init({
 });
 ```
 
+### `table_use_colgroups`
+
+{{ site.requires_5_5v }}
+
+This option allows you to use colgroups/cols for columns widths for new tables created by the table plugin. Tables inserted by other means, or already existing tables, will not be changed.
```

Comment:
```suggestion
This option adds `colgroups` and `cols` elements to new tables for specifying column widths. Existing tables and tables added using `setContent` or paste are unaffected.
```

Reply:
Should be `colgroup` and `col` elements I think? Those are the actual names.

Reply:
I fixed this in https://github.com/tinymce/tinymce-docs/pull/1679/commits/0cee6158f721b9b97725ed81ec56a53b03ab8dba after I realised that :slightly_smiling_face: